RELEASE/scripts/autoscend.ash

```diff
@@ -362,12 +364,21 @@ boolean LX_burnDelay()
 				return true;
 			}
 		}
+		if(wannaBackup)
```

Comment:
downside of putting this here is it won't backup if there's no delay to burn

Reply:
I thought it won't force a delay zone but will adventure wherever it was going to anyways. It will still backup at next combat. I'm thinking that seems too random and would be better to default to noob cave like you suggested. Thoughts?

Reply:
it needs to be noob cave because you might be adventuring somewhere that doesn't have 100% combat rate where you really want the monster (lobsterfrogmen)

Reply:
Believe LFM is the only one we care about for forcing adventure into Noob Cave. Added that but honestly feels like I just wedged it in there. I have been seeing monster replacement handling that I image is what was your original suggestion to update - hence the feeling of wedging it in and going against the framework in place.

Getting late here and not sure I'm making any sense. Will be ascending by week end and will start actually testing this. Thanks for your feedback

Reply:
Did look closer at our existing replace monster code. Wasn't able to figure out a good way to use what we had with backup camera. Existing replace handling relies on the new monster being a random one from the same zone. Where as with backup camera we know what the new monster will be. 

So all in all don't feel like I wedged this code in anymore.